src/GroundWaterMmfMod.F90

```diff
@@ -264,27 +262,18 @@ subroutine LATERALFLOW  (NoahmpIO, ISLTYP,WTD,QLAT,FDEPTH,TOPO,LANDMASK,DELTAT,A
   REAL,    PARAMETER :: PI = 3.14159265 
   REAL,    PARAMETER :: FANGLE = 0.22754493   ! = 0.5*sqrt(0.5*tan(pi/8))
 
-#ifdef MPP_LAND
-! create halo'ed local copies of tile vars
-  landmask_h(ims:ime, jms:jme) = landmask
-  area_h(ims:ime, jms:jme)     = area
 
-  nx = ((ime-ims) + 1) + 2      ! include halos
-  ny = ((jme-jms) + 1) + 2      ! include halos
-  
-  !copy neighbor's values for landmask and area
-  call mpp_land_com_integer(landmask_h, nx, ny, 99)
-  call mpp_land_com_real(area_h, nx, ny, 99)
-
-  itsh=max(its,1)
-  iteh=min(ite,global_nx)
-  jtsh=max(jts,1)
-  jteh=min(jte,global_ny)
+
+#ifdef MPP_LAND
+    itsh=ims
+    iteh=ime
+    jtsh=jms
+    jteh=jme
 #else
-  itsh=max(its-1,ids)
-  iteh=min(ite+1,ide-1)
-  jtsh=max(jts-1,jds)
-  jteh=min(jte+1,jde-1)
+    itsh=its
+    iteh=ite
+    jtsh=jts
+    jteh=jte
```

Comment:
these don't change answers for non-parallel runs?

Reply:
In the LIS group, we actually did something similar to these indices when adding MMF parallel into Noah-MP-4.0.1.

See this code change:
https://github.com/NASA-LIS/LISF/pull/1418/files#diff-7737fa95c763c3f4c44635b9bf558d8a063138b94eb0e208fc2209d2ed65ef3e

Scroll down to Lines 329 to 338 in the new code.

We haven't merged this PR yet, but will be doing it soon.